Source/Revit.IFC.Export/Exporter/WallSweepExporter.cs

```diff
@@ -1,4 +1,4 @@
-﻿//
+//
 // BIM IFC library: this library works with Autodesk(R) Revit(R) to export IFC files containing model geometry.

```

Comment:
Note that currently the master branch is not up-to-date.  Look at the 24 branch for the latest.  We will be updating the master branch to avoid future confusion.

Reply:
FYI: in 23.3.1 it had not been updated.

![image](https://github.com/Autodesk/revit-ifc/assets/55183821/f7dfbc19-3c7c-4070-bb6a-fa2c45f74f65)